CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # OHHTTPStubs — CHANGELOG
 
 ## Master
+* Fixed `URLRequest.ohhttpStubs_httpBody` function in Swift 3 and 4.
```

Comment:
Please add 2 spaces after the period (so that the rendered markdown forces a newline-in-same-paragraph and makes the links go to a dedicated line)

Reply:
Also I think GitHub's markdown parser prefers having empty lines after section titles like `## Master`